src/main/groovy/jenkins/automation/utils/ScmUtils.groovy

```diff
@@ -27,6 +27,9 @@ class ScmUtils {
                     if (use_versions && version!=null) {
                         branch("*/tags/$version")
                     }
+                    else {
+                        branch "master"
```

Comment:
groovy "poetic mode". You  can drop the parenthesis if there is only one argument.